src/state_title_screen.c

```diff
@@ -33,9 +33,11 @@ SaveMetadata gSaveSlotMetadata[4] = {
 };
 
 u8 gSaveSlotHasData[4] = {TRUE, TRUE, TRUE, TRUE};
-s32 D_80077A28 = 0;
-s32 D_80077A2C = 0;
-s32 D_80077A30 = 0;
+
+s32 TitleScreen_PressStart_Alpha = 0; // the transparency of "PRESS START" text
```

Comment:
If I'm understanding the code correctly, it's actually _opacity_. (so 0 would be transparent, 255 would be fully visible). Transparency would work the other way around (with 100% or 255 being fully transparent)